### PR TITLE
docs: update images using latest UI screenshots

### PR DESCRIPTION
**Pull request by NicolaeUnleash**
Updating images in the documentation using the latest UI screenshots

https://linear.app/unleash/issue/1-6/update-images-in-the-docs

### CODE REVIEW

**Review by ivarconr — APPROVED** [review]
wow, this is a lot of work. Fantastic. Maybe we can automate this at some point with the frontend tests in the future somehow? wdyt @Tymek ?

**Comment by Tymek** [comment]
> wow, this is a lot of work. Fantastic. Maybe we can automate this at some point with the frontend tests in the future somehow? wdyt @Tymek ?

We can improve it and work on some tools to speed it up. In my opinion switching to an automatic generation is not the way to go. We didn't update screenshots in a while - this should be accounted for when working on new features but it wasn't. I'd like to have a system of "snapshots" warning about "hey, you changed something that is on a screenshot in docs - here is the link"

**Review by Tymek — COMMENTED** [review]
`website/static/img/constraints-add-to-strategy.png` is new, right? Is it in use in docs?

**Review by Tymek — APPROVED** [review]

**Comment by NicolaeUnleash** [comment]
We should wait with this until we have the 4.15 release

File: website/docs/how-to/how-to-add-strategy-constraints.md [hunk 1]
```diff
@@ -2,11 +2,9 @@
 title: How to add strategy constraints
 ---
 
-:::info Availability
-Strategy constraints are available to Unleash Pro and Enterprise users.
-:::
+:::info Availability Strategy constraints are available to Unleash Pro and Enterprise users. :::
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-add-strategy-constraints.md:5]: This is something that prettier does that breaks the layout, so we need to add some spaces like this:

```suggestion
:::info Availability 

Strategy constraints are available to Unleash Pro and Enterprise users. 

:::
```

File: website/docs/how-to/how-to-add-strategy-constraints.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Every strategy will have button labeled "add constraints" when viewed in the adm
 
 Refer to [the _constraint structure_ section of the strategy constraints reference](../advanced/strategy-constraints.md#constraint-structure) for a thorough explanation of the fields.
 
-From the "Context Field" dropdown, select the context field you would like to constrain the strategy on.
+From the "Context Field" dropdown, select the context field you would like to constrain the strategy on. @Thomas we need to update the text here
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-add-strategy-constraints.md:27]:
```suggestion
This is a template suggestion! Don't accept this, but this is how it'll work.
```

File: website/docs/how-to/how-to-create-and-assign-custom-project-roles.md [hunk 3]
```diff
@@ -36,10 +36,10 @@ Custom project role creation is a pretty straightforward process and requires ar
 To assign a custom project role to a user:
 1. Navigate to the project you want to assign the user a role in.
     ![The steps to navigate to a project: use the 'projects' navigation item and select your project.](/img/assign-cpr-step-1.png)
-2. Navigate to the project's _access_ page.
+2. Navigate to the project's _access_ tab.
     ![A project overview with the 'access' tab highlighted.](/img/assign-cpr-step-2.png)
 3. This step depends on whether the user has already been added to the project or not:
-    - If the user has already been added to the project, select the new role you want to give them from the dropdown menu next to their name.
+    - If the user has already been added to the project, click on the edit icon coresponding with its line and from the overlay that will show up select the new role you want to assign it from the dropdown and save the changes.
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-create-and-assign-custom-project-roles.md:42]:
```suggestion
    - If the user has already been added to the project: Use the edit button in the user's row. Select the role you want to assign to the user, and then save your changes.
```

File: website/docs/how-to/how-to-create-and-assign-custom-project-roles.md [hunk 4]
```diff
@@ -36,10 +36,10 @@ Custom project role creation is a pretty straightforward process and requires ar
...
         ![A list of users with access to the current project. To the right of each user is a dropdown input labeled role.](/img/assign-cpr-step-3a.png)
-    - If the user _hasn't_ been added to the project, add them via the 'add user' form. Select the role you want to give them from the role field.
+    - If the user _hasn't_ been added to the project, add them using the button 'Assing user/group'. From the overlay that will show up select the user, assign it a role and save the changes. Now you should be able to see the new user in the table.
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-create-and-assign-custom-project-roles.md:44]:
```suggestion
    - If the user _hasn't_ been added to the project, add them using the button labeled 'Assign user/group'. Use the form to select the user and the desired role, and then save your changes. The user will appear in the users table.
```

File: website/docs/how-to/how-to-schedule-feature-releases.mdx [hunk 5]
```diff
@@ -35,7 +35,7 @@ This guide assumes that you've got the following:
 
 To schedule a feature release via the UI:
 1. Add the desired activation strategy to the feature
-2. Open the constraint creator by using the "add constraint" button
+2. Open the constraint creator by using the "Add custom constraint" button
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-schedule-feature-releases.mdx:38]: Goes back to a previous comment: should we capitalize these? I'm a bit on the fence 💁🏼

```suggestion
2. Open the constraint creator by using the "add custom constraint" button
```

File: website/docs/user_guide/environments.md [hunk 6]
```diff
@@ -40,7 +40,7 @@ Note that in order to enable an environment for a feature toggle, you must first
 
 ### Step 1: Enable new environments for your Project
 
-Navigate to the project and choose the “environments” tab.
+Navigate to the project and choose the “Environments” tab.
```
Comment (thomasheartman) [inline website/docs/user_guide/environments.md:43]: Unsure about this change too

File: website/docs/user_guide/environments.md [hunk 7]
```diff
@@ -51,7 +51,7 @@ From the “feature toggle view” you will now be able to configure activation
 ![A feature toggle strategies tab showing three different environments, of which one is active. The UI displays data about the currently selected environment, ](/img/environments_strategies.png 'Add strategy configuration per environment')
 
 ### Step 3: Create environment specific API keys
-
+@Thomas - please double-check if the image si ok with the  text bellow
```
Comment (thomasheartman) [inline website/docs/user_guide/environments.md:54]: Sure, that looks about right, but the alt text will need to be updated.

```suggestion
```

File: website/docs/user_guide/environments.md [hunk 8]
```diff
@@ -51,7 +51,7 @@ From the “feature toggle view” you will now be able to configure activation
...
 In order for the SDK to download the feature toggle configuration for the correct environment you will need to create an API token for a defined environment.
 
 ![An API key creation form. The form's fields are "username", "token type", "project", and, crucially, "environment". The development environment is selected.](/img/environments_api_keys.png 'Create Environment specific API Keys')
```
Comment (thomasheartman) [inline website/docs/user_guide/environments.md:57]:
```suggestion
![An API token creation form and how to get there in the Admin UI. The steps are: navigate to Configure > API access > New API token.](/img/environments_api_keys.png 'Create Environment specific API Keys')
```

File: website/docs/user_guide/projects.md [hunk 9]
```diff
@@ -37,7 +37,7 @@ From the top-line menu – click on “Projects”
 
 ![The Unleash admin UI with the "Projects" nav link in the top bar highlighted.](/img/projects_button.png)
 
-The available projects will now be listed. To create a new Project – choose the “Add new project”
+The available projects will now be listed. To create a new Project – choose the “New project”
```
Comment (thomasheartman) [inline website/docs/user_guide/projects.md:40]:
```suggestion
The UI shows the currently available projects. To create a new project, use the “new project” button.
```

File: website/docs/user_guide/quickstart.md [hunk 10]
```diff
@@ -260,6 +260,7 @@ Follow steps outlined in the [Run Unleash with Docker](#run-unleash-with-docker)
    ![The Unleash UI showing a dropdown menu under the Configure menu entry. The dropdown menu's API Access option is highlighted and you're told to navigate there.](/img/api_access_navigation.png 'Navigate to the API access page.')
 
    Next, create an API key with these details
+   @Thomas - token type needs to be updated based on the new screen
```
Comment (thomasheartman) [inline website/docs/user_guide/quickstart.md:263]:
```suggestion
```

File: website/docs/user_guide/quickstart.md [hunk 11]
```diff
@@ -260,6 +260,7 @@ Follow steps outlined in the [Run Unleash with Docker](#run-unleash-with-docker)
    ![The Unleash UI showing a dropdown menu under the Configure menu entry. The dropdown menu's API Access option is highlighted and you're told to navigate there.](/img/api_access_navigation.png 'Navigate to the API access page.')
 
    Next, create an API key with these details
+   @Thomas - token type needs to be updated based on the new screen 
 
    - **name:** proxy-key (this can be whatever you want)
    - **token type:** client
```
Comment (thomasheartman) [inline website/docs/user_guide/quickstart.md:266]:
```suggestion
   - **what do you want to connect** server-side SDK (CLIENT)
```

Comment (thomasheartman) [reply website/docs/user_guide/quickstart.md:266]: @NicolaeUnleash is this what you meant?

File: website/docs/user_guide/technical-debt.md [hunk 12]
```diff
@@ -15,8 +15,8 @@ A toggle being (potentially) stale, does not affect how it performs in your appl
 
 
 ## The technical debt dashboard
-
-In order to assist with removing unused feature toggles, Unleash provides a technical debt dashboard in the management-ui. You can find it by clicking on “Advanced” in the top-line menu then choose _Reporting_ in the  dropdown menu.
+@Thomas - is this technical debt dashboard still the correct naming? (wouldn't make more sens to updat it Health? in the entire page)
```
Comment (thomasheartman) [inline website/docs/user_guide/technical-debt.md]: How about a compromise? Change the title to "Technical debt and project health"? Or maybe just "project health"? 🤔

File: website/docs/user_guide/unleash_overview.md [hunk 13]
```diff
@@ -22,7 +22,7 @@ Before you can connect your application to Unleash you need a Unleash server. Yo
 ### System Overview {#system-overview}
 
 ![A visual overview of an Unleash system as described in the following paragraph.](/img/unleash-architecture.svg 'System Overview')
-
+@Thomas - we need to review the text here, i updated the image (new: the orange text and one more option/setup)
```
Comment (thomasheartman) [inline website/docs/user_guide/unleash_overview.md]: I did some stuff in 7f6794ae21f56d67407d8a33f540d15d7d17d7e0 which I think should cover it. Let me know if you disagree 😄

File: website/docs/user_guide/user-management.md [hunk 14]
```diff
@@ -11,10 +11,10 @@ You can add new users to Unleash in `Admin > Users`.
     ![A visual representation of the current step: the Unleash Admin UI with the steps highlighted.](/img/user_admin_list_button.png)
 
 
-2. To add a new user to your Unleash instance simply click the "Add user" button:
+2. To add a new user to your Unleash instance simply click the "New user" button:
```
Comment (thomasheartman) [inline website/docs/user_guide/user-management.md:14]:
```suggestion
2. To add a new user to your Unleash instance, use the "new user" button:
```

File: website/docs/user_guide/user-management.md [hunk 15]
```diff
@@ -11,10 +11,10 @@ You can add new users to Unleash in `Admin > Users`.
...
     ![The Unleash users page with the 'add new user' button being pointed to.](/img/user_admin-add-user.jpg)
 
-3. Fill out the required fields about the user in the modal. You need to choose which role the new user should have on the "root level". E.g. giving the user an Editor role will allow the user to create a new project.
+3. Fill out the required fields about the user in the create user page. You need to choose which role the new user should have on the "root level". E.g. giving the user an Editor role will allow the user to create a new project.
```
Comment (thomasheartman) [inline website/docs/user_guide/user-management.md:17]:
```suggestion
3. Fill out the required fields in the "create user" form. Refer to the [global roles overview](./rbac.md#standard-roles) for more information on roles.
```

**Review by thomasheartman — COMMENTED** [review]
Great work on this and lots of nice things you've caught 🙌🏼 

I think I've either updated everything you've requested or asked for clarification if I'm not sure or if I disagree.

Just a few things to note in general (I think I saw a few of these):
1. Try and avoid words like "click" (consider "select" or "use"; you can't click if you don't have a mouse 🐭 ) and "see" (not everyone has functioning eyesight 👀 )
2. When updating (and changing images), make sure the alt text still applies or update it if it doesn't 😄

File: website/docs/how-to/how-to-create-and-assign-custom-project-roles.md [hunk 16]
```diff
@@ -22,9 +22,9 @@ It takes about three steps to create custom project roles:
 
 1. Navigate to the custom project roles page by using the admin menu (the gear symbol) and navigating to users.
     ![A visual representation of the current step: the Unleash Admin UI with the steps highlighted.](/img/create-cpr-step-1.png)
-2. Navigate to the "project roles" tab.
+2. Navigate to the "Project roles" tab.
```
[the remark this answers is not in the document]
Comment (NicolaeUnleash) [reply website/docs/how-to/how-to-create-and-assign-custom-project-roles.md:25]:
```suggestion
2. Navigate to the "project roles" tab.
```

File: website/docs/user_guide/environments.md (hunk 6 above)
Comment (NicolaeUnleash) [reply website/docs/user_guide/environments.md:43]:
```suggestion
Navigate to the project and choose the “environments” tab.
```

Comment (thomasheartman) [inline website/docs/user_guide/environments.md:43]:
```suggestion
Navigate to the project and choose the “environments” tab.
```

File: website/docs/user_guide/projects.md [hunk 17]
```diff
@@ -93,6 +93,6 @@ All available projects are available from the drop-down menu.
 
 ## Change project for an existing feature toggle {#change-project-for-an-existing-feature-toggle}
 
-If you want to change which project a feature toggle belongs to, you can change that from the feature toggle's configuration page. Under the _settings_ tab, choose the _project_ option and choose the new project from the dropdown menu.
+If you want to change which project a feature toggle belongs to, you can change that from the feature toggle's configuration page. Under the "Settings" tab, choose the _project_ option and choose the new project from the dropdown menu.
```
Comment (thomasheartman) [inline website/docs/user_guide/projects.md:96]:
```suggestion
If you want to change which project a feature toggle belongs to, you can change that from the feature toggle's configuration page. Under the "Settings" tab, choose the _project_ option and choose the new project from the dropdown menu.
```
```suggestion
If you want to change which project a feature toggle belongs to, you can change that from the feature toggle's configuration page. Under the "Settings" tab, choose the _project_ option and choose the new project from the dropdown menu.
```
```suggestion
If you want to change which project a feature toggle belongs to, you can change that from the feature toggle's configuration page. Under the "Settings" tab, choose the _project_ option and choose the new project from the dropdown menu.
```
```suggestion
If you want to change which project a feature toggle belongs to, you can change that from the feature toggle's configuration page. Under the _settings_ tab, choose the _project_ option and choose the new project from the dropdown menu.
```

**Review by thomasheartman — APPROVED** [review]
Super duper! 🦸🏼 🙌🏼

**Pull request merged**